tests/acceptance/30_generic_methods/file_ensure_key_value_present_in_ini_section.cf

```diff
@@ -52,13 +52,14 @@ ${base_text_down}";
 ${name_2}=${value_2}
 ${base_text_up}";
 
-# Third test: ensure that if the line already exists, it will be modified
+# Third test: ensure that if the line already exists, it will be modified (must have an end section to work with cfengine <3.10)
     "section_3"         string => "section_test1";
     "name_3"            string => "name";
     "value_3"           string => "value";
     "new_value_3"       string => "newvalue";
-    "reference_3"       string => "${base_text_up}
-${name_3}=${new_value_3}";
+    "reference_3"       string => "[${section_3}]
+${name_3}=${new_value_3}
+[endfile]";
```

Comment:
We should not require this, we have an exeption in stdlib to handle the behavior change.

Reply:
it does not work on cfengine 3.7 without this

Reply:
https://tracker.mender.io/browse/CFE-2263?jql=text%20~%20%22select_region%22

Reply:
it actually fails from 3.6.3 to 3.8 according to ticket

Reply:
How could this test past before in this case?

Reply:
we only have agent in 3.6.2 and 3.10 if i'm correct

Reply:
We have 3.6.5 in 3.1, and we had 3.7 in 3.2/4.0

Reply:
you are perfectly right, I was testing with a vanilla 3.7